src/arewefluentyet/source.py

```diff
@@ -8,10 +8,60 @@ def parse_date(input):
 
 
 class Source:
-    def __init__(self, path):
+    current_revision: str | None = None
```

Comment:
I'm not familiar with the syntax, but why the assignment to None if it's already part of the constructor method? Or am I misunderstanding this code completely?

Reply:
```suggestion
    current_revision: str | None
```